source/geometry_model/chunk.cc

```diff
@@ -175,7 +117,6 @@ namespace aspect
       return std::map<std::string,types::boundary_id>();
     }
 
-
```

Comment:
Oh, are you willing to re-add these empty lines? We typically keep 3 empty lines between functions.